client/asset/btc/electrum.go

```diff
@@ -19,6 +19,11 @@ import (
 	"github.com/btcsuite/btcd/chaincfg/chainhash"
 )
 
+const (
+	needBTCElectrumVersion = "4.5.5"
+	needLTCElectrumVersion = "4.2.2"
+)
```

Comment:
There's a `MinNetworkVersion` field in the `BTCCloneCFG`. Can we use that so that these versions are set in their respective packages?

Reply:
Oh, that's an int and you need a string.

Reply:
Added `MinElectrumVersion` and firo is working.. but not the txhistory, as firo does not to have the `onchain_history` method. https://github.com/decred/dcrdex/compare/4bc7ecb667be572b2580ca816621dd7c314b6009..202269e3de3c5220f38e2080b2c8ff6a5d0d38c0

Reply:
I was unable to build the firo electrum client, unsure how to fix and just used the binary